packages/kit-haptics/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { ActionsDefinition } from '@rise-tools/react'
+import * as Haptics from 'expo-haptics'
+import z from 'zod'
+
+import type { ImpactAction, NotificationAction, SelectionAction } from './server'
+
+const ImpactActionPayload = z.object({
+  style: z.nativeEnum(Haptics.ImpactFeedbackStyle).optional(),
+})
+
+const NotificationActionPayload = z.object({
+  type: z.nativeEnum(Haptics.NotificationFeedbackType).optional(),
+})
+
+type HapticsActions = ActionsDefinition<ImpactAction | NotificationAction | SelectionAction>
+
+export const useHapticsActions = (): HapticsActions => {
+  return {
+    'haptics/impact': {
```

Comment:
I thought you wanted to use the package name as the prefix? So the action would be `@rise-tools/kit-haptics/impact`

Reply:
Yeah, I likely forgot about it, but it makes sense! 😆

Reply:
Updated!